docs/core/config/auth/databases/passwd.md

```diff
@@ -51,8 +51,8 @@ the fields to be imported need to be explicitly defined.
 userdb passwd {
   fields_import_all = no
   fields {
-    uid = %{passwd:uid:vmail}
-    gid = %{passwd:gid:vmail}
+    uid = %{passwd:uid}
```

Comment:
%{passwd:uid|default('vmail')}?

Reply:
passwd always has uid and gid, so I don't see any point giving a default